administrator/language/en-GB/com_templates.ini

```
@@ -24,6 +24,7 @@ COM_TEMPLATES_BUTTON_RENAME_FILE="Rename File"
 COM_TEMPLATES_BUTTON_RESIZE="Resize"
 COM_TEMPLATES_BUTTON_UPLOAD="Upload"
 COM_TEMPLATES_CHECK_FILE_OWNERSHIP="Check file ownership"
+COM_TEMPLATES_CHILD_SUCCESS="Child template succesfully created."
```

kill the successfully - it is redundant (and spelt wrong)

```suggestion
COM_TEMPLATES_CHILD_SUCCESS="Child template created."
```